spec/features/user_groups_spec.rb

```diff
@@ -22,7 +22,10 @@
     org = create(:user_group)
     login_as(admin)
 
-    visit edit_user_group_path(org)
+    visit user_groups_path
+    find("a[aria-label='#{t('links.aria.edit', name: org.name)}']").click
```

Comment:
I set it up with links that are just "Edit" and "Delete", when running tests, i found that there were multiple user_groups on the page.